### PR TITLE
re-dump schema with rails 5.1 to avoid future diffs

### PR DESCRIPTION
**Pull request by grosser**
this is pretty ugly, but it's what we have in prod ... and that should be our dev starting point ...

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -12,527 +12,529 @@
...
+    t.string "source_url"
+    t.index ["created_by"], name: "index_builds_on_created_by"
+    t.index ["git_sha"], name: "index_builds_on_git_sha", unique: true
+    t.index ["project_id"], name: "index_builds_on_project_id"
   end
 
-  create_table "commands", force: :cascade do |t|
-    t.text     "command",    limit: 10485760
+  create_table "commands", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.text "command", limit: 16777215
```
Comment (sandlerr) [inline db/schema.rb]: does that match your dev database or our prod one?

**Comment by sandlerr** [comment]
`DEFAULT CHARSET=utf8` seems wrong, don't we want `utf8mb4` everywhere in mysql? is there a way to get this working in postgresql and sqlite too?

File: db/schema.rb (hunk 1 above)
Comment (grosser) [inline db/schema.rb]: prod

**Comment by grosser** [comment]
I'll see what I need to change to get this loadable on postgres/sqlite ...

File: db/schema.rb [hunk 2]
```diff
@@ -12,527 +12,529 @@
...
     t.datetime "deleted_at"
-    t.string   "source",     null: false
-    t.index ["project_id", "branch"], name: "index_webhooks_on_project_id_and_branch", length: { branch: 191 }, using: :btree
-    t.index ["stage_id", "branch"], name: "index_webhooks_on_stage_id_and_branch", length: { branch: 191 }, using: :btree
+    t.string "source", null: false
+    t.index ["project_id", "branch"], name: "index_webhooks_on_project_id_and_branch"
+    t.index ["stage_id", "branch"], name: "index_webhooks_on_stage_id_and_branch"
   end
 
+  add_foreign_key "deploy_groups", "environments"
```
Comment (sandlerr) [inline db/schema.rb]: any idea why these foreign keys were missing? hope we don't have any constraint violations in our database

Comment (grosser) [inline db/schema.rb]: idk ... will add a new migration to drop them ... already tried a week ago or so and that did not work out :/

**Comment by sandlerr** [comment]
shame about `id: :integer` noise, would have thought that is the default

**Comment by grosser** [comment]
they changed it to bigint in 5.1 ... so all the old ones are integer ... we
could migrate em ... but don't see a reason why ... we won't hit 32bit
anytime ...

**Comment by grosser** [comment]
manually updated `output` and `object` column to what was specified in the migrations ... prod mysql multiplies them by 4 because we use multibyte ...

**Comment by grosser** [comment]
added Rakefile fixes so load/dump will not produce diff anymore

**Pull request merged**